.tmp/drop/status

```diff
@@ -0,0 +1,2 @@
+1494417747482
```

Comment:
There's no reason to include `.tmp` directory. This folder contains build artifacts.